packages/core/src/tools/internal-tools/implementations/delegate-to-url-tool.ts

```
@@ -0,0 +1,284 @@
+            .describe(
+                'Optional session ID for maintaining conversation state across multiple delegations to the same agent'
+            ),
+        timeout: z
+            .number()
+            .optional()
+            .default(30000)
+            .describe('Request timeout in milliseconds (default: 30000)'),
+    })
+    .strict();
+
+type DelegateToUrlInput = z.infer<typeof DelegateToUrlInputSchema>;
```

ESLint

[error] Use z.output instead of z.infer for better type inference with Zod schemas. z.output includes transformations while z.infer may miss them.

(no-restricted-syntax)